src/Features/Onboarding.php

```diff
@@ -574,7 +574,11 @@ public static function has_woocommerce_support( $theme ) {
 		}
 
 		foreach ( $themes as $theme ) {
-			$directory = new \RecursiveDirectoryIterator( $theme->theme_root . '/' . $theme->stylesheet );
+			$stylesheet_file = $theme->theme_root . '/' . $theme->stylesheet;
+			if ( ! file_exists( $stylesheet_file ) ) {
+				continue;
+			}
```

Comment:
So, just to clarify; it looks like if a stylesheet is not present, the theme is more-or-less ignored for the purposes of checking for WooCommerce support? Assuming that's the behavior we want, this looks good to me :+1:

Reply:
@joelclimbsthings Yup, that's the case. This is somewhat edge case since people usually don't structure a theme directory in such a way and even the [official doc](https://developer.wordpress.org/themes/advanced-topics/child-themes/) recommends putting a child theme at the top level. Thank you for the review 👍